libs/beacon/include/beacon/entropy.hpp

```diff
@@ -43,6 +43,11 @@ struct Entropy
     return *reinterpret_cast<uint64_t const *>(entropy.pointer());
   }
 
+  operator bool() const
+  {
+    return !entropy.empty();
```

Comment:
Actually, it can be argued that the entropy is valid only if it has 32 bytes.

Reply:
you're right